actions/src/pull_requests/claCheck.js

```diff
@@ -112,7 +112,10 @@ const checkSheet = async (auth) => {
         core.setFailed('No data found.');
       } else {
         core.info(`Checking if ${PR_AUTHOR} has signed the CLA`);
-        const hasUserSignedCla = flatRows.includes(PR_AUTHOR);
+        const hasUserSignedCla = flatRows.some(
+          item => item.toLowerCase() === PR_AUTHOR.toLowerCase()
```

Comment:
Maybe `user` would be a better variable name instead of `item`?
Since the array consists of users who have signed the CLA.

Reply:
Done.